packages/backend/src/models/role.js

```diff
@@ -85,6 +87,59 @@ class Role extends Base {
         });
     });
   }
+
+  async deleteWithPermissions() {
```

Comment:
I got an error while deleting role of deleted user:
snackbar "Failed while deleting" 500 error on the response and backend console error